.github/workflows/cmake-build-editor-linux.yml

```
@@ -24,20 +36,12 @@ jobs:
         conan upload "*" -r=artifactory -c
         source ./build/Release/generators/conanbuild.sh
 
     - name: Configure CMake for release
-      run: cmake --preset conan-release
-
-    - name: Build Release
-      run: cmake --build --preset conan-release -j
-
-    - name: Configure CMake Debug
-      run: cmake --preset conan-debug
+      run: cmake --preset conan-${{ matrix.build_type == "Debug" && "debug" || "release" }}
```

actionlint (1.7.4)

got unexpected character '"' while lexing expression, expecting 'a'..'z', 'A'..'Z', '_', '0'..'9', ''', '}', '(', ')', '[', ']', '.', '!', '<', '>', '=', '&', '|', '*', ',', ' '. do you mean string literals? only single quotes are available for string delimiter

(expression)